src/pattern-library/components/patterns/MoleculePatterns.js

```diff
@@ -98,6 +101,58 @@ export default function MoleculePatterns() {
           </PatternExample>
         </PatternExamples>
       </Pattern>
+
+      <Pattern title="Modal container">
+        <p>
+          The <code>modal</code> pattern positions and sizes a modal container
+          appropriately based on viewport size.
+        </p>
+        <PatternExamples>
+          <PatternExample details="Responsive Modal container positioning and sizing">
+            <div>
+              <LabeledButton
+                variant="primary"
+                onClick={() => setShowExample1(true)}
+              >
+                Show example
+              </LabeledButton>
+              <div
+                className="hyp-u-overlay"
+                style={
+                  showExample1 ? 'visibility:visible' : 'visibility:hidden'
+                }
+              >
+                <div className="hyp-modal">
```

Comment:
Ah - I think I'm beginning to appreciate how the molecules / utility classes / components fit together now. This starting to come together nicely.

Reply:
Well, that's good! They are quite useful at least in the pattern-library context at present...